docs/customer-portal/cp-data-management.md

```diff
@@ -4,27 +4,6 @@ The **Database** allows you to create custom persistent data storage. A datastor
 
```

Comment:
A datastore can be specific to a single customer or global (used with any customer).
Is that the beginning please remove that